### PR TITLE
Prepare for version 0.1.2

### PR DESCRIPTION
**Pull request by hweng**
@awead, can we push the new version to rubygem? My UCSD hydra app depends on the new version. Thanks!

### CODE REVIEW

**Comment by hweng** [comment]
All rspec tests and rubocop passed on my local. Not sure why it failed on travis-ci, any idea?

**Comment by mjgiarlo** [comment]
@hweng It looks like it's because Rubocop 0.38.0 breaks rubocop-rspec. There's a PR in to fix that bug [here](https://github.com/bbatsov/rubocop/pull/2936/files) but you can see it's not yet merged, so there's no 0.38.1 available to fix you up. You can either point at Rubocop 0.37.2, or wait on 0.38.1, or tweak `.rubocop.yml` to remove the offending line (temporarily).

**Comment by hweng** [comment]
@mjgiarlo , thanks for the suggestion. It looks good now after point at Rubocop 0.37.2.

File: Gemfile [hunk 1]
```diff
@@ -4,6 +4,6 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'byebug'
-gem 'rubocop', require: false
+gem 'rubocop', "~> 0.37.2", require: false
```
Comment (awead) [inline Gemfile:7]: It's presently working under 0.38, so we shouldn't need this.

File: spec/push_document_spec.rb [hunk 2]
```diff
@@ -25,16 +25,6 @@
...
-  describe '@providerUpdatedDateTime' do
-    subject do
-      valid = described_class.new(uri)
-      valid.add_contributor(name: 'Job', email: 'joe@joe.com')
-      valid.title = 'Some title'
-      valid
-    end
-    it { is_expected.to be_valid }
-  end
-
```
Comment (awead) [inline spec/push_document_spec.rb:37]: Why is this being removed?

**Comment by awead** [comment]
In general, we increment gem versions in their own commit, ex. 07e8ce10c0ade302b6d33defa97de0883aee2f49. So for this PR, there wouldn't be any any changes to the version, and you'd save that change for a subsequent commit.

**Comment by awead** [comment]
@hweng 0.2 is out, so I'm going to close this PR.

**Pull request closed without merging**